internal/common/labels_test.go

```diff
@@ -75,4 +75,14 @@ var _ = Describe("AddAppLabels", func() {
 		labels := obj.GetLabels()
 		Expect(labels[AppKubernetesManagedByLabel]).To(Equal("ssp-operator"))
 	})
+
+	It("adds required labels", func() {
```

Comment:
After the change, this test can be removed, because it will be covered by other tests.

Reply:
Done.